app/controllers/api_controller.rb

```diff
@@ -122,11 +122,11 @@ def set_access_control_headers
   end
 
   def base_config
-    Api::Settings.base
+    ManageIQ::API::Settings.base
```

Comment:
@jrafanie OIC! Yes, it could do, if that's a concern. I'd have to use the non-compact style, i.e.

```ruby
module ManageIQ
  module API
    class BaseController < ApplicationController
# etc....
```

vs 

```ruby
ManageIQ::API::BaseController
```

In order for that to work, unless I'm mistaken